support/itemUtils.ts

```diff
@@ -73,15 +73,14 @@ export function getViewProperties(config: ApplicationConfig): any {
   };
 }
 
-export function createMap(item: PortalItem): IPromise<WebMap | WebScene> {
+export function createMap(item: PortalItem, appItem: any): IPromise<WebMap | WebScene> {
```

Comment:
@kellyhutchins can we change to this?

```
interface Proxy {
// whatever proxy things we need.
}

export function createMap(item: PortalItem, appProxies?: Proxy[]):
```

I think this is better so someone can just pass the appProxies if they need to. The `?` will make appProxies optional